### PR TITLE
fix tensorrt output varible reshape

### PR DESCRIPTION
**Pull request by shangzhizhou**
### PR types
Bug fixes

### PR changes
Others

### Describe
fix bug of PR https://github.com/PaddlePaddle/Paddle/pull/29308
remove ernie fc padded shape

### CODE REVIEW

**Review by cryoco — DISMISSED** [review]
LGTM

File: paddle/fluid/inference/tensorrt/convert/fc_op.cc [hunk 1]
```diff
@@ -142,9 +142,77 @@ class FcOpConverter : public OpConverter {
     TensorRTEngine::Weight bias{nvinfer1::DataType::kFLOAT,
                                 static_cast<void*>(bias_data),
                                 static_cast<size_t>(bias_num)};
+    PADDLE_ENFORCE_EQ(
+        x_num_col_dims == 1 || x_num_col_dims == 2, true,
+        platform::errors::InvalidArgument(
+            "Wrong x_num_col_dims param of op mul. Paddle-TRT FC converter "
+            "expects x_num_col_dims is either 1 or 2, but got %d",
+            x_num_col_dims));
```
Comment (cryoco) [inline paddle/fluid/inference/tensorrt/convert/fc_op.cc]: 这个要不要拿到opteller里

**Review by cryoco — DISMISSED** [review]

File: paddle/fluid/inference/tensorrt/convert/fc_op.cc (hunk 1 above)
Comment (shangzhizhou) [reply paddle/fluid/inference/tensorrt/convert/fc_op.cc]: done

**Review by cryoco — DISMISSED** [review]
LGTM

**Review by cryoco — APPROVED** [review]

**Pull request merged**